jerry-ext/debugger/debugger-tcp.c

```diff
@@ -152,6 +165,7 @@ jerryx_debugger_tcp_send (jerry_debugger_transport_header_t *header_p, /**< tcp
   JERRYX_ASSERT (jerry_debugger_transport_is_connected ());
 
   jerryx_debugger_transport_tcp_t *tcp_p = (jerryx_debugger_transport_tcp_t *) header_p;
+  jerryx_socket_size_t message_to_send_length = (jerryx_socket_size_t) message_length;
```

Comment:
(minor) This is a bit long variable name. Let's look for something simpler but still descriptive. Could be `remaining_bytes`, as there is already a variable `sent_bytes`.